### PR TITLE
W-14900257: branch to a dummy workspace when an error occurred

### PR DESCRIPTION
**Pull request by llibarona**

### CODE REVIEW

File: als-server/js/src/main/scala/org/mulesoft/als/server/protocol/configuration/ClientAlsInitializeParams.scala [hunk 1]
```diff
@@ -37,7 +38,8 @@ object ClientAlsInitializeParams {
           internal.workspaceFolders.map(_.map(_.toClient)).map(_.toJSArray).getOrElse(null).asInstanceOf[js.Any],
         rootPath = internal.rootPath.orUndefined,
         initializationOptions = internal.initializationOptions.collect { case js: js.Object => js }.orUndefined,
-        hotReload = internal.hotReload.orUndefined
+        hotReload = internal.hotReload.orUndefined,
+        hotReload = internal.maxFileSize.orUndefined
```
Comment (BlockLucas) [inline als-server/js/src/main/scala/org/mulesoft/als/server/protocol/configuration/ClientAlsInitializeParams.scala:42]: duplicated line without ammend

File: als-server/jvm/src/test/scala/org/mulesoft/als/server/lsp4j/Lsp4jLanguageServerImplTest.scala [hunk 2]
```diff
@@ -128,8 +129,11 @@ class Lsp4jLanguageServerImplTest extends AMFValidatorTest with ChangesWorkspace
         """.stripMargin
 
       for {
-        _      <- executeCommandIndexDialect(server)(dialectPath, dialectContent)
-        config <- wm.getWorkspace(dialectPath).flatMap(_.getConfigurationState)
+        _ <- executeCommandIndexDialect(server)(dialectPath, dialectContent)
+        config <- wm
+          .getWorkspace(dialectPath)
+          .map { case wcm: WorkspaceContentManager => wcm }
```
Comment (BlockLucas) [inline als-server/jvm/src/test/scala/org/mulesoft/als/server/lsp4j/Lsp4jLanguageServerImplTest.scala:135]: is this change intended?

File: dependencies.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 version=6.1.0-SNAPSHOT
-amf=5.4.8-RC.1
+amf=5.5.0-INTERNAL-UNHANDLEDEXCEPTION
```
Comment (BlockLucas) [inline dependencies.properties:2]: Is there an official snapshot / stable version to use here?

File: als-common/shared/src/main/scala/org/mulesoft/als/configuration/MaxSizeException.scala [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package org.mulesoft.als.configuration
+
+import amf.core.client.scala.exception.AmfUnhandledException
+
+case class MaxSizeException(url: String, maxSize: Int)
+    extends AmfUnhandledException(s"maximum file size of $maxSize exceeded")
```
Comment (mansillafederico) [inline als-common/shared/src/main/scala/org/mulesoft/als/configuration/MaxSizeException.scala:6]: Whats is the unit of $maxSize? I mean Mb, Kb, characters...

Comment (BlockLucas) [reply als-common/shared/src/main/scala/org/mulesoft/als/configuration/MaxSizeException.scala:6]: those are bytes

**Review by BlockLucas — APPROVED** [review]
